localMatchApp/sources/Login.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Login.h"
+
+using namespace std;
+
+Login::Login ( ) {
+  // Connect to database.
+}
+
+Login::~Login ( ) {
+  // Disconnect to database.
+}
+
+Login::connectionMessage Login::signUp ( const string i_user, const string i_password,
+                                         const string i_repeatedPassword ) {
+  // Send information to database.
+  string userInDatabase = "";
+
+  if ( hasCorrectFormat ( i_password ) && hasCorrectFormat ( i_repeatedPassword ) ) {
+    if ( i_user.compare ( userInDatabase ) == 0 ) {
```

Comment:
Refactor this, too many nested if/else statements

Reply:
Done.